### PR TITLE
Add ChatGLM3 support

### PR DESCRIPTION
**Pull request by SakuraYM**
only change the convert tool name to add glm3 support

### CODE REVIEW

File: src/xfastertransformer/__init__.py [hunk 1]
```diff
@@ -23,13 +23,13 @@
 
 _import_structure = {
     "automodel": ["AutoModel"],
-    "tools": ["LlamaConvert", "ChatGLMConvert", "ChatGLM2Convert", "OPTConvert", "BaichuanConvert"],
+    "tools": ["LlamaConvert", "ChatGLMConvert", "ChatGLM23Convert", "OPTConvert", "BaichuanConvert"],
```
Comment (Duyi-Wang) [inline src/xfastertransformer/__init__.py:26]: I think providing three API `ChatGLM2Convert`, `ChatGLM3Convert`,`ChatGLM23Convert` is more friendly for user and forward compatible, even they are one thing.

Comment (Duyi-Wang) [reply src/xfastertransformer/__init__.py:26]: > the suggestion given by pujiang is not to duplicate the code, just modify the name to quickly support it.

No need to duplicate the code...
`ChatGLM23Convert` as the parent classes, set name `chatglm23` in config.ini.
The others are child class. different is the __init__().

Comment (SakuraYM) [reply src/xfastertransformer/__init__.py:26]: a new model type in config.ini need duplicate src/models/x.cpp, it is not a passable parameter.

Comment (Duyi-Wang) [reply src/xfastertransformer/__init__.py:26]: > a new model type in config.ini need duplicate src/models/x.cpp, it is not a passable parameter.

If you don't want to inherit multiple classes in C++, why can't the higher-level Python provide three api? As long as the generated config file is the same.

**Review by pujiang2018 — APPROVED** [review]

**Comment by pujiang2018** [comment]
merged.
@Duyi-Wang could you pls further refactor it to make more sense?

**Pull request merged**